back-end/services/DictionaryService.js

```diff
@@ -1,27 +1,27 @@
-const DictionaryModel = require.main.require('./models/Dictionary');
+const Dictionary = require.main.require('./models/Dictionary');
```

Comment:
What is `require.main.require`?

Reply:
In order to prescribe absolute paths and not relative